ydb/core/etcd/events.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#pragma once
+
+#include "proto.h"
+#include <ydb/library/actors/core/event_local.h>
+#include <ydb/library/actors/core/events.h>
+
+namespace NYdb::NEtcd {
+
+struct TEvPrivate {
+    // Event ids
+    enum EEv : ui32 {
+        EvRangeResponse = EventSpaceBegin(NActors::TEvents::ES_PRIVATE),
```

Comment:
I'm not sure, but may be we need to create new ID group for etcd events such [as](https://github.com/pavelbezpravel/ydb/blob/etcd-kv-service/ydb/core/base/events.h#L177)